util/Fs.cpp

```diff
@@ -252,7 +252,10 @@ ResourcePressure Fs::readRespressure(const std::string& path) {
     // aggr 316016073
     // some 0.00 0.03 0.05
     // full 0.00 0.03 0.05
-    OCHECK(lines.size() == 3);
+    if (lines.size() != 3) {
+      short_lived_exception exception_var( "(short lived process ) read Mempressure");
+      throw &exception_var;
```

Comment:
Throwing a pointer to something on the stack is probably not a good idea. Although it might still work, this is undefined behavior. Prefer doing this instead:
```
throw std::runtime_error("readMempressure could not read filesystem");
```